src/09-word-problems.js

```diff
@@ -80,18 +125,38 @@ function getCartTotal(products) {}
     compareLocations(address1, address2);
     //> "Same city."
  */
-function compareLocations(address1, address2) {}
+function compareLocations(address1, address2) {
+  if ((address1.city === address2.city) && (address1.state === address2.state) && (address1.zip === address2.zip) && (address1.street === address2.street)) {
+    return "Same building.";
+  } else {
+      if ((address1.city === address2.city) && (address1.state === address2.state) && (address1.zip === address2.zip) && (address1.street !== address2.street)) {
```

Comment:
With long chained statements and so many operators it is a good idea to break your lines into multi-line statements after the ` && ` or ` || ` operators.  It makes things easier to read.